lib/timers.js

```diff
@@ -215,6 +215,15 @@ function setInterval(callback, repeat, arg1, arg2, arg3) {
   return timeout;
 }
 
+ObjectDefineProperty(setInterval, customPromisify, {
```

Comment:
Im unsure if setInterval should ever be passed to `promisify`, and its usage would be different from every other function you pass to `promisify`... because `setInterval` will return `{ [Symbol.asyncIterator]: Function }` rather than a promise

Reply:
Probably not, since it doesn't return promises.